src/Riok.Mapperly.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,6 @@
 #nullable enable
+
+Riok.Mapperly.Abstractions.MapEnumValueAttribute
```

Comment:
You can add these lines to the shipped file directly, we don't really use the unshipped file.